Build failure: unnnamed-sdvx-clone

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

Try to build unnamed-sdvx-clone from the repository.

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/295713608

### Relevant log output

```console
Running phase: unpackPhase
unpacking source archive /nix/store/3zwm1cyn23pp0hkw6yvks0b3awmfv18j-source
source root is source
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/share/doc/USC -DCMAKE_INSTALL_INFODIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/strip -DCMAKE_RANLIB=/nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/ranlib -DCMAKE_AR=/nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/pk4mipq9as76k4dnywzflahzqnwmvv77-unnamed-sdvx-clone-0.6.0 -DUSE_SYSTEM_CPR=ON -DCMAKE_BUILD_TYPE=Release
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/dc6bahp3f5af2rxz3pal9m3kp4vx4rpy-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Freetype: /nix/store/15755cxaiv750fs6p2qwwq3r01g94b37-freetype-2.13.3/lib/libfreetype.so (found version "2.13.3")
-- Found ZLIB: /nix/store/8dr41ckl7sbv2v5aim3csxzql03z68hk-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Found SDL2: /nix/store/3r1cplqpqfxc3j3my2ma0l311rs932qy-sdl2-compat-2.32.54-dev/include
-- Found PNG: /nix/store/srcfig7bzw7br814yy8iivq5al07cdii-libpng-apng-1.6.46/lib/libpng.so (found version "1.6.46")
-- Found JPEG: /nix/store/vvbavgrybcbd1iq04953jpgpg8hpdkdb-libjpeg-turbo-3.0.4/lib/libjpeg.so (found version "62")
-- Found Vorbis: /nix/store/sllnjqkmc527amm0lw1vj7zjm5kd6xdc-libogg-1.3.5-dev/include
-- Found OGG: optimized;/nix/store/jqwh458rb4jnwdg45yx3jz9ss25n19is-libogg-1.3.5/lib/libogg.so;debug;/nix/store/jqwh458rb4jnwdg45yx3jz9ss25n19is-libogg-1.3.5/lib/libogg.so
-- Found LibArchive: /nix/store/63lan55wm3n1h5r9rn7gbihpzfsnkmi1-libarchive-3.7.8-lib/lib/libarchive.so (found version "3.7.8")
-- Performing Test Iconv_IS_BUILT_IN
-- Performing Test Iconv_IS_BUILT_IN - Success
-- Found Iconv: built in to C library
Enabling crash dumps
CMake Deprecation Warning at third_party/lua-5.3.5/CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


-- Found CURL: /nix/store/056kf3y6n9q5i7gdiynwdj6wrmb7b4kx-curl-8.12.1/lib/libcurl.so (found version "8.12.1")
CMake Deprecation Warning at third_party/discord-rpc/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


CMake Error at /nix/store/np45wjc61sy6d021dnrgyizga8f859k5-cmake-3.31.6/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find X11 (missing: X11_X11_INCLUDE_PATH X11_X11_LIB)
Call Stack (most recent call first):
  /nix/store/np45wjc61sy6d021dnrgyizga8f859k5-cmake-3.31.6/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/np45wjc61sy6d021dnrgyizga8f859k5-cmake-3.31.6/share/cmake-3.31/Modules/FindX11.cmake:676 (find_package_handle_standard_args)
  Graphics/CMakeLists.txt:54 (find_package)


-- Configuring incomplete, errors occurred!
```

### Additional context

_No response_

### System metadata

- system: `"x86_64-linux"`
 - host os: `Linux 6.6.87, NixOS, 25.05 (Warbler), 25.05.20250423.8a2f738`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.2`
 - nixpkgs: `/nix/store/2ab9ljvsgkmn3zizflwf61207a23q6q1-source`

### Notify maintainers

@Sakooooo 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/un/unnamed-sdvx-clone/package.nix
+++ b/pkgs/by-name/un/unnamed-sdvx-clone/package.nix
@@ -16,12 +16,13 @@
   curl,
   libcpr,
   rapidjson,
+  libX11,
+  libGL,
   writeShellScriptBin,
   makeDesktopItem,
   lib,
   copyDesktopItems,
 }:
-
 stdenv.mkDerivation (finalAttrs: {
   pname = "unnamed-sdvx-clone";
   version = "0.6.0";
@@ -54,6 +55,8 @@ stdenv.mkDerivation (finalAttrs: {
     curl
     libcpr
     rapidjson
+    libX11
+    libGL
   ];
 
   cmakeFlags = [

